### ISSUE TITLE
To do: outputs

### ISSUE THREAD

**Issue by robj411** [ITHIM/ITHIM-R#56]
- [x] Travel summary tables

- [ ] PA summary tables

- [ ] AP summary tables

- [ ] Injuries summary tables

- [x] Parameter summary tables

- [ ] AP+PA health summary tables

- [ ] Injury health summary tables

- [ ] AP health

- [ ] PA health

**Comment by usr110** [comment]
Update on `travel summary tables` - which in theory is completed for four case studies.

The recent change was made using @JDWoodcock suggestions

- Please put only 1 decimal place
- Time-  minutes per person per day
- Distance – km per person per day

Can be seen at: https://github.com/ITHIM/ITHIM-R/blob/master/summary_tables.html

In case of `Delhi` you might notice that the `trip duration (mins)` is too low. I have just investigated this issue and found that the high number of people without trips is creating such low numbers. Just to remind you, these numbers are per person now (counting both people with and without trips).

In total, there are `9162` participants, out of which only `5243` have trips, which means `3919` have no trips at all. 

> length(unique(io$delhi$trip_scen_sets$participant_id))
[1] 5243

> length(unique(io[[city]]$synth_pop$participant_id))
[1] 9162

@rahulatiitd @robj411

**Comment by JDWoodcock** [comment]
thanks can I download the table from the link? I would expect for Delhi low trip rates amongst women are what is going on, illustrating why extra analysis will be needed bespoke on top of basic tables

**Comment by usr110** [comment]
> 
> 
> thanks can I download the table from the link? I would expect for Delhi low trip rates amongst women are what is going on, illustrating why extra analysis will be needed bespoke on top of basic tables

Use this then: https://raw.githubusercontent.com/ITHIM/ITHIM-R/master/summary_tables.html and then save the html page locally.

**Comment by JDWoodcock** [comment]
magic

**Comment by JDWoodcock** [comment]
could we have the modes in the same order for each table with the case specific modes at the end. Could we sum up public transport (or at least group them so they are together).
Could we have totals for each column
thanks

**Comment by usr110** [comment]
Please wait a bit. Just realized I had copied the new expanded SP's trip dataset (which was sampled for 1000 trips before) at an incorrect path. Now rerunning the model and outputting the tables.

**Comment by rahulatiitd** [comment]
> thanks can I download the table from the link? I would expect for Delhi low trip rates amongst women are what is going on, illustrating why extra analysis will be needed bespoke on top of basic tables

@JDWoodcock @robj411 per capita distance for Bangalore totals (for all modes) to 5.2 and for Delhi 8.1, this means partly the explanation for Delhi's low duration per capita may be because it is in hours not minutes?Can @usr110 check this?  London Travel demand survey reports 12.6 km per capita. 
The duration per capita for Accra is somewhat strange. It totals to about 100 minutes per capita and For Sao Paulo it is 95. In London this is 63 (1 hour per capita is a well known phenomenon). But not sure what we can do with it.  

It is possible that Accra is 'adults-only' so the average travel time of the population is getting inflated? @leandromtg can confirm. 

I am going to work on 'trip-only' tables- average trip duration, trip length etc and see if anything pops out.

**Comment by JDWoodcock** [comment]
I agree Delhi's duration is clearly wrong. Distances are plausible. 
Duration for Accra & SP are plausible to me. We had high values in Switzerland too (1 hour is rough guide).
But I agree we do need to be clear about age structure we are looking at.

**Comment by rahulatiitd** [comment]
50 minutes per capita in Buses for Accra is not at all possible. London with all public transport modes (trains and buses) spends 30 minutes per capita. On an average Accra residents cannot be spending that long in a bus. But again, we cannot do much about it except making sure that the denominator is not a subset of population.

**Comment by JDWoodcock** [comment]
Accra is from a time use survey not travel survey so estimates are likely different.  Rahul can you check if time use tend to produce longer estimates?

Get Outlook for Android<https://aka.ms/ghei36>




On Fri, Apr 12, 2019 at 9:08 AM +0100, "rahulatiitd" <notifications@github.com<mailto:notifications@github.com>> wrote:


50 minutes per capita in Buses for Accra is not at all possible. London with all public transport modes (trains and buses) spends 30 minutes per capita. On an average Accra residents cannot be spending that long in a bus. But again, we cannot do much about it except making sure that the denominator is not a subset of population.

**Comment by usr110** [comment]
> Delhi's low duration per capita may be because it is in hours not minutes?Can @usr110 check this?

No, that's not the case any more. Now the unit is minutes in Delhi too. You may double check it yourself. Read: "inst/extdata/local/delhi/trips_delhi.csv"

**Comment by rahulatiitd** [comment]
> > Delhi's low duration per capita may be because it is in hours not minutes?Can @usr110 check this?
> 
> No, that's not the case any more. Now the unit is minutes in Delhi too. You may double check it yourself. Read: "inst/extdata/local/delhi/trips_delhi.csv"

Are these numbers not updated in 'io' dataset?

**Comment by usr110** [comment]
Due to large SP dataset, I could not update the `io` object - sorry about that. You may run the `multi_city_script` till L169 first - this will save the object in the right place, and then run the `summary_table.Rmd` document.

**Comment by usr110** [comment]
Argh. Sorry it seems the `stage_duration` is in hours. I thought I had fixed it.

**Comment by JDWoodcock** [comment]
@rahulatiitd Accra is from a time use survey not travel survey so estimates are likely different. But looking at this http://atususers.umd.edu/papers/BOSESHARP.pdf (for US setting) table 2 not clear that time use produces long estimates

**Comment by rahulatiitd** [comment]
It could be a resolution problem-- here is how the travel time by bus looks like in accra
 [1]  60  30  30  60  45 105 113  60  30  60  90  60  90  60  60  15  75  30  30  60  30  75  30  90  30  30  60  60  30  30  60  30  30  30  20  10  25  30  40  60  55
  [42] 100  60  65  60  30  30  15  15  30  60  60  30  30  25  50  25  30  30  60  40  60  60  40  60  30  50  60 120  50  60  50  60  60  60  30  60  30  20  60  30  25
  [83]  30  60  40  60  50 110  60 120  10  60 120  60  60  90  40  40  15  15  15  60  10  60  30  90  45  60  30  30  60  40  60  30  45  60  60  60  30  60  70  60  60
 [124]  90  60 120  50  30  60  60  10  50  10  55  10  50  30  60  60  30  90 120  55  85  60  90  20  30  30  30  30  30  90  30  90  30  60  60  30  50  40  40  60  30
 [165]  60  90  50  60  20  30  30  45  60  40  40  60  40  45 105  40  40  30  30  40  30  20  20  40  60 120  40  15  15  45 105  60  60 120  30  60  60  60 120  10  10
 [206]  20  20  30  30  20  60  45  50  55  60  45  75  60  75  60  20  30  50  30  90  15  15  10  20  60  60  45  45  60  30  45  60  30  30  60  60  10  10  10  20  20
 [247]  80 140  15  30  15  30  10  30  20  30  30  15  30  30  20  25  25  25  35  95 115  30  30  90  30  60  30  60  30  60  30  15  60  40  30  10  60  60  90  20  30
 [288]  30  30  50  60  30  60  45  10  50  10  20  20  30  30  30  30  30  30  30  30  45  60  90  10  60  60  90  60  30  30  30  60  30  90  60  30  60  30  60  20  20
 [329]  30  15  60  60  60  30  30  60 120  60  40  60  60  60  60  60  15  30  30  30  40  30  30  60  30  90  40  40  60  60  30  30  15  45  30  40  40  15  30  60  90
 [370]  60  15  45  30  90  30  60  30  30  20  20  20  30  60  60  60  60  30  60  30  45  60  60  60  90  60  30  90  40  60  90  60  60  25  55  60  25  60  30  30  60
 [411]  50  80  30  30  30  90  40  10  40  20  50  60  60  60  40  10  20  40  60  60  20  20  60  60  10  70 115  60 120  30  60  30  40  60  60  25  55  60  90  20  50
 [452]  60  60 120  55  85  60 120 140  35  30  30  15  30  45  10  50  15  45  45  10  30  15  60 120  60 120  30  49  30  60  60  60  15  45  45  30  45  25  60  75  30
 [493]  30  20  20  50  60  60  60  30  30  40  15  40  15  15  60  60  20  30  15  30  20  60  20  30  45  45  35  60  60  60 120  60  90  30  75  60 120  30  45  30  30
 [534]  90 120  60  90  30  90  60  60  60  60  60  90  60  30  30  20  60  35  20  20  30  30  15  30  30  90  15  35  50 110  30  30  60  30  60  90  60  30  60  20  40
 [575]  60  15  75  90  60 120 135  60  30  60  45  45  20  60  60  55  10  60  55 115 175  60  50  50  60  45  60  90  50  60  60  60  60  60 120  60 120  45  35  95  30
 [616]  90  30  30  60  60  60  60  25  30  30  30  60  15  45  60  30  45 105  30  80  45  30  90  30  90  30  85  60  15  45  30  45  50 110  30  20  30  90  30  20  30
 [657]  30  30  40 100 125  60  75  60  60  50  60  90  45 105  30  45  50  40  30  60 120 180  30  30  10  55  30  90 120  10  25  20  20  60  20  30  30  30  40 100 120
 [698]  40  70  40  10  50  60  45  15  55  30  60  60  40  70  30  10  40  60 120  60  20  60  75  30  60  30  60  75  60  75  30  30  60  60  15  35  20  25  25  60  60
 [739]  15  20  20  60  60  60  60  35  30  90  35  15  15  30  30  30  90  60 120 180  60  60  60  90  45  20  80 140  60 105  30  60  60  30  60  30  30  60  45 105 113
 [780]  60  30  60  90  60  90  60  60  15  75  30  30  60  30  75  30  90  30  30  60  60  30  30  60  30  30  30  20  10  25  30  40  60  55 100  60  65  60  30  30  15
 [821]  15  30  60  60  30  30  25  50  25  30  30  60  40  60  60  40  60  30  50  60 120  50  60  50  60  60  60  30  60  30  20  60  30  25  30  60  40  60  50 110  60
 [862] 120  10  60 120  60  60  90  40  40  15  15  15  60  10  60  30  90  45  60  30  30  60  40  60  30  45  60  60  60  30  60  70  60  60  90  60 120  50  30  60  60
 [903]  10  50  10  55  10  50  30  60  60  30  90 120  55  85  60  90  20  30  30  30  30  30  90  30  90  30  60  60  30  50  40  40  60  30  60  90  50  60  20  30  30
 [944]  45  60  40  40  60  40  45 105  40  40  30  30  40  30  20  20  40  60 120  40  15  15  45 105  60  60 120  30  60  60  60 120  10  10  20  20  30  30  20  60  45
 [985]  50  55  60  45  75  60  75  60  20  30  50  30  90  15  15  10

**Comment by JDWoodcock** [comment]
suggest not to fill github issue list with so many numbers. Do you mean it is rounded up? Sounds plausible. We could also check with Fidelia what she thinks.

**Comment by rahulatiitd** [comment]
i was going to delete this right after you read

**Comment by rahulatiitd** [comment]
its deleted now. yes, i meant rounded off not by respondents, but by design of questionnaire..

**Comment by JDWoodcock** [comment]
I agree - something has to fill the time. But that US study didnt seem to find this

**Comment by usr110** [comment]
Fixed the unit (hours to mins) bug in trip_duration for Delhi, and then rerun the model and created the summary_table again. Now the numbers look much higher. 

![image](https://user-images.githubusercontent.com/10958628/56032002-e1c6a100-5d18-11e9-9cb7-1fce21fbd81b.png)

You may access the summary_table yourself by looking at: https://raw.githubusercontent.com/ITHIM/ITHIM-R/master/summary_tables.html

@rahulatiitd

**Comment by JDWoodcock** [comment]
thanks Bangalore doesnt look right though  - 20 minute travel per person per day at baseline looks very low. 
@usr110 could you add totals to the table?

**Comment by usr110** [comment]
@JDWoodcock just looked at the numbers, and similar to `Delhi`, `Bangalore` too has a high number of people without trips (almost double), which is making the numbers too low (the denominator was `total people` meaning count both `people with trips` and `people without trips`).

![image](https://user-images.githubusercontent.com/10958628/56033426-83032680-5d1c-11e9-840b-88e6af594207.png)

Now you may read the `summary_table.html` again to see these numbers (only for `duration` now, but for both `everyone` and `people with trips` as denominator, in separate tables).

**Comment by JDWoodcock** [comment]
thanks @usr110 . @rahulatiitd i think you need to check the travel survey on this and see what is going on. Even the people with trips are doing less than 40 minutes per day

**Comment by rahulatiitd** [comment]
> thanks @usr110 . @rahulatiitd i think you need to check the travel survey on this and see what is going on. Even the people with trips are doing less than 40 minutes per day

yes, I will
